public/static/docs/command-reference/config.md

```diff
@@ -114,18 +114,9 @@ for more details.) This section contains the following options:
   > option, properly transforming paths relative to the current working
   > directory into paths relative to the config file location.
 
-- `cache.protected` - make DVC-tracked files read-only. Possible values are
-  `true` or `false` (default). Run `dvc checkout` after changing the value of
-  this option for the change to go into effect.
-
-  Due to the way DVC handles linking between the data files in the cache and
-  their counterparts in the <abbr>workspace</abbr>, it's easy to accidentally
-  corrupt the cached file by editing or overwriting it. Turning this config
-  option on forces you to run `dvc unprotect` before updating a file, providing
-  an additional layer of security to your data.
-
-  We highly recommend enabling `cache.protected` when `cache.type` is set to
-  `hardlink` or `symlink`.
+- `cache.protected` - obsoleted. When using `hardlink` or `symlink` as
+  `cache.type`, they will automatically be read-only. Use `dvc unprotect`
+  before updating a file.
 
 - `cache.type` - link type that DVC should use to link data files from cache to
   the workspace. Possible values: `reflink`, `symlink`, `hardlink`, `copy` or a
```

Comment:
should we mention that we need to `dvc checkout --relink` to enforce the change (to make it safe)?

Reply:
@shcheklein To enforce the change after we've made it default? Not sure it is worth noting in the docs. People will re-clone or re-checkout things anyway and that comment will be useful for a few days that people are migrating from old version to the new one.